.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - main
     paths:
-      - "docs/sources/**"
```

Comment:
We should keep this PR focused and not include the changes in the Markdown files. I've opened https://github.com/grafana/explore-profiles/pull/59 to take care of it.

Reply:
I must have ran a linter at some point. I don't recall changing this file 😬 reverted in https://github.com/grafana/explore-profiles/pull/25/commits/4bffbdf3848d5a3197122fb1150ec86e80173a75